src/components/AddUsersModal/index.jsx

```diff
@@ -102,15 +102,27 @@ class AddUsersModal extends React.Component {
       greeting: formData['email-template-greeting'],
       closing: formData['email-template-closing'],
     };
-    const hasTextAreaEmails = !!formData['email-addresses'];
-    const emails = hasTextAreaEmails ? formData['email-addresses'].split(/\r\n|\n/) : formData['csv-email-addresses'];
-    const { validEmails } = validateEmailAddresses(emails);
-    options.user_emails = validEmails;
+
+    // Validate email addresses from both the text area and the CSV file and
+    //   submit to the backend for emails to be sent and/or error to be displayed
+    const emails = [];
+    if (formData['email-addresses'] && formData['email-addresses'].length) {
+      formData['email-addresses'].split(/\r\n|\n/).forEach((email) => {
```

Comment:
tiny nit: could use the `emails.push(...formData['email-addresses'].split(/\r\n|\n/))` syntax here (and below) to save a few lines and avoid the `forEach` 🤷‍♂️